modules/util/JSUtil.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Gets object's values.
+ * @param {Object} object
+ * @return {Array<Object>}
+ */
+export function getValues(object) {
```

Comment:
Is this "es-shims" included through some other lib ? I don't see it declared anywhere in package.json ? Should I add it ? If I see correctly it adds few dependencies. I will probably use a Map or something instead and have this one removed.